### PR TITLE
Add tests for PersistentNetNamesDisable

### PR DESCRIPTION
**Pull request by kokesak**
Test for PersistentNetNamesDisable actor.

### CODE REVIEW

**Comment by pirat89** [comment]
leapp-ci build

File: repos/system_upgrade/el7toel8/actors/persistentnetnamesdisable/tests/test_persistentnetnamesdisable.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    pci1 = PCIAddress(domain="ax", bus="bx", function="cx", device="dx")
+    pci2 = PCIAddress(domain="ay", bus="by", function="cy", device="dy")
+    interface = [Interface(name="eth0", mac="52:54:00:0b:4a:6d", vendor="adh",
+                           driver="as", devpath="rrx", pci_info=pci1),
+                 Interface(name="eth1", mac="52:54:00:0b:4a:6a", vendor="adh",
+                           driver="az", devpath="rry", pci_info=pci2)]
+    current_actor_context.feed(PersistentNetNamesFacts(interfaces=interface))
+    current_actor_context.run()
+    assert 'inhibitor' in current_actor_context.consume(Report)[0].flags
+
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/actors/persistentnetnamesdisable/tests/test_persistentnetnamesdisable.py]: missing test 1 eth & 1+ non-eth

**Comment by kokesak** [comment]
I added one test for one ethX and one not_ethX

**Comment by pirat89** [comment]
It looks like broken CI again.

File: repos/system_upgrade/el7toel8/actors/persistentnetnamesdisable/tests/test_persistentnetnamesdisable.py (hunk 1 above)
Comment (pirat89) [reply repos/system_upgrade/el7toel8/actors/persistentnetnamesdisable/tests/test_persistentnetnamesdisable.py]: @kokesak Do not forget to "resolve conversation" when you resolve the requested changes.

**Comment by pirat89** [comment]
leapp-ci build

**Comment by pirat89** [comment]
leapp-ci build

**Comment by jmikovic** [comment]
please rebase

**Comment by vojtechsokol** [comment]
leapp-ci build

**Comment by bocekm** [comment]
@kokesak, the rebase isn't correct. Please talk to me or @AloisMahdal (I talked to him) to do it the right way.
Basically you need to add https://github.com/oamg/leapp-repository to your git config. So, make sure you have something like this in .git/config:
```
[remote "origin"]
	url = git@github.com:kokesak/leapp-repository.git
	fetch = +refs/heads/*:refs/remotes/origin/*
[branch "master"]
	remote = upstream
	merge = refs/heads/master
[remote "upstream"]
	url = git@github.com:oamg/leapp-repository.git
	fetch = +refs/heads/*:refs/remotes/upstream/*
```
And then do:
```
$ git checkout master
$ git pull
$ git checkout test
$ git rebase master
$ git push --force
```

**Comment by AloisMahdal** [comment]
Yeah, during our session I completely forgot to sync with usptream; we rebased against your fork's master which is kinda pointless...

BTW to add upstream origin you don't need to touch .git/config (and I'd avoid that as it's easy to mess up things there).

    git remote add upstream git@github.com:oamg/leapp-repository.git
    git branch master -u upstream

then the commands Michal posted should work.

Hint: A nice command to review local/remote branching & the tracking relations using:

    git branch -avv

**Comment by bocekm** [comment]
The rebase is correct now, tests pass, @pirat89 did a review, so I'm merging it.

**Comment by AloisMahdal** [comment]
@kokesak :congratulations:

**Pull request merged**